eng/Versions.props

```diff
@@ -9,8 +9,7 @@
     <RepositoryUrl>https://github.com/dotnet/dotnet-monitor</RepositoryUrl>
     <VersionPrefix>8.0.0</VersionPrefix>
     <PreReleaseVersionLabel>rtm</PreReleaseVersionLabel>
-    <!-- Disable final version kind until merged into release branch. -->
-    <!--<DotnetFinalVersionKind>release</DotnetFinalVersionKind>-->
+    <DotnetFinalVersionKind>release</DotnetFinalVersionKind>
```

Comment:
Maybe let's keep the comment here? We'll want to revert this for 9 previews

Reply:
This is targeting `release/8.x`; the comment remains in the `main` branch.